vpr/src/route/fibonacci_heap.h

```diff
@@ -0,0 +1,38 @@
+#ifndef _FIBONACCI_HEAP_H
+#define _FIBONACCI_HEAP_H
+
+#include "heap_type.h"
+
+class FibonacciHeap : public HeapInterface {
+  public:
+    FibonacciHeap();
```

Comment:
I assume this is the encapsulation of the traditional VPR heap.

Unless I'm miss-understanding, I believe the current heap is a binary heap (not a Fibonacci heap).

Reply:
You are probably right.  I'll rename

Reply:
Current heap is a binary heap, not fibonnaci

Reply:
Ya, this has been renamed.